srcpkgs/mosquitto/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'mosquitto'
 pkgname=mosquitto
-version=1.6.0
+version=1.6.3
 revision=2
```

Comment:
revision must be bumped

Reply:
why? It's a new version so usually the revision would be reset to 1.

Reply:
Didn't see its an update. Revision needs to be set to 1